Huu

codyelj opened the pull request:
https://github.com/ResearchKit/ResearchKit/issues/1453#issue-838902922

akshay-yadav-apple committed:
updates

Hengyu committed:
Use implied answer format for cells

srinathtm-apple committed:
Use wheel date picker style (#1447)

* Use wheel date picker style

stevemoser committed:
Fix broken Apple Forums link

erik-apple committed:
Permission Type Updates (#1454)

* Ensure that the slider's colors match the view's tint

* New visual style for request permission step

* Add a new notifications permission type

* Add a new motion activity permission type

* Update ORKRequestPermissionButton.m

* Forward declare button

* Update ResearchKit/Common/ORKPermissionType.m

Co-authored-by: joeylabarck-apple <81833193+joeylabarck-apple@users.noreply.github.com>

* Update ResearchKit/Common/ORKPermissionType.h

Co-authored-by: joeylabarck-apple <81833193+joeylabarck-apple@users.noreply.github.com>

* Update ResearchKit/Common/ORKMotionActivityPermissionType.h

Co-authored-by: joeylabarck-apple <81833193+joeylabarck-apple@users.noreply.github.com>

* Update ResearchKit/Common/ORKPermissionType.h

Co-authored-by: joeylabarck-apple <81833193+joeylabarck-apple@users.noreply.github.com>

* Update ResearchKit/Common/ORKPermissionType.h

Co-authored-by: joeylabarck-apple <81833193+joeylabarck-apple@users.noreply.github.com>

* Add missing import

* Add missing import

Co-authored-by: joeylabarck-apple <81833193+joeylabarck-apple@users.noreply.github.com>

gavirawson-apple committed:
Clip step image (#1457)

erik-apple committed:
Updates to the predefined range of motion task (#1459)

erik-apple committed:
Replace deprecated method call

erik-apple committed:
Merge pull request #1461 from erik-apple/update-deprecated-method-in-keychain

Update deprecated method call

erik-apple committed:
Silence warning about nullable return types

erik-apple committed:
Merge pull request #1462 from erik-apple/nullable-return-type

Silence warning about nullable return types

Pariecemckinney-apple committed:
Merge pull request #1338 from Hengyu/hengyu

Use implied answer format for cells

Pariecemckinney-apple committed:
Merge pull request #1448 from stevemoser/patch-6

Fix broken Apple Forums link

cbaker6 committed:
Set ORKTaskViewController nav background color (#1469)

Pariecemckinney-apple committed:
initial commit

Pariecemckinney-apple committed:
Merge pull request #1471 from Pariecemckinney-apple/pmckinney/taskVCDeprecationWarningFix

Fix ORKTaskViewController deprecation warning

aplummer-apple committed:
Update project library search paths to compile on apple silicon (#1479)

I believe this line was in here to resolve issues with an early beta of xcode 11 and is unnecessary now

Pariecemckinney-apple committed:
IOS15 Fixes (#1487)

Pariecemckinney-apple committed:
Point RK to IOS 13 (#1499)

akshay-yadav-apple committed:
merging stable back to main
Resolving Merge conflicts
Bugfixes
Cleanup

akshay-yadav-apple committed:
merge stable into main
resolve merge conflicts
Bugfixes
Cleanup
merge main back to stable

akshay-yadav-apple committed:
Merge branch 'stable' into 'main' for housekeeping

akshay-yadav-apple committed:
README update

akshay-yadav-apple committed:
marker

SwiftArtery committed:
Prevent isPasscodeStoredInKeychain crash, correct warnings (#1505)

* fix: prevent isPasscodeStoredInKeychain from causing a crash

* fix: prevent nskeyedunarchiver warnings, add missing classes

GameCubeGBA commented:
> [#1453 (comment)](https://github.com/ResearchKit/ResearchKit/issues/1453#issue-838902922)

Can you please fix the title of this

GameCubeGBA commented:
Also please explain your changes

Pariecemckinney-apple committed:
2.1.1 Release

-Bug fixes
-New ORKAccuracyStroopStep
-AudioStep now allows for audio recording 
-New Styles for ORKDontKnowButton
-General layout and UI improvements

NYMZA commented:
SIA HU HEKA MEHAN ZULU HULU AZULI ANANU MANUPEREGAL

cbaker6 committed:
propagate tintColor if window not available

cbaker6 committed:
only check for tintColor once in viewControllerForStep

cbaker6 committed:
Fix tintColor for ORKWebViewStepViewController buttons

cbaker6 committed:
make suggested changes

louie-apple committed:
Point ResearchKit.podspec to 2.1.0

louie-apple committed:
remove deprecated ORKConsent files

louie-apple committed:
Merge pull request #1530 from ResearchKit/cocoapod_version_bump

Point ResearchKit.podspec to 2.1.0

louie-apple committed:
Merge pull request #1531 from ResearchKit/cocoapod_fix

remove deprecated ORKConsent files

louie-apple committed:
Merge pull request #1519 from cbaker6/fixTintColor

Propagate tintColor if window not available

Pariecemckinney-apple committed:
Release 2.2.8 

-Bug fixes and enhancements
-New helper method (ORKViewTintColor) for setting tintColor
-New “displayUnit” property added to ORKAnswerFormat
-Updates in ORKNormalizedReactionTimeResult’s secure coding (breaks compatibility with previous archives).

ronzilla-apple committed:
Public release/2.2.9 (#1541)

* Address issue where there was no option to "Save Results" if task is cancelled (Issue #1536/1540)
* Address an issue where ORKTaskViewController setting tintColor on stepViewController views broke ORKStepViewController's addResult:

* Fixed order issue of tone audiometry steps
* Include missing ORKSpeechRecognition header export (Issue #1534)
* Switch to contentEdgeInsets API in ORKContinueButton to fix an issue where button title wouldn't always fit onscreen (on iOS 16 during long press) (Issue #1533)
* Remove min/max values for ORKCatalog "Numeric with Display Unit"

* xcconfig file updates for version and driving ORKCatalog bundleID
* Addressing compiler warnings
* Added unit tests for init'ing taskViewController with restoration data
* Adding task restoration feature to ORKCatalog's taskViewController presentation

Pariecemckinney-apple committed:
Public release/2.2.10 (#1548)

Fix ORKNavigationContainerView so that it does not call didMoveToWindow from tintColorDidChange
Bumped version to 2.2.10

Pariecemckinney-apple committed:
Public release/2.2.12 (#1555)

- Fixed `ORKHealthKitQuantityTypeAnswerFormat`issues
- Improved `ORKReviewStep` initialization
- Introduced a new property called `shouldAutomaticallyAdjustImageTintColor` on `ORKStep` to automatically adopt dark mode version of an image
- Added `ORKSESQuestionResult` to Table View Providers
- ORKCatalog Improvements
- ORKImageSelectionView improvements
- Bumped version to 2.2.12

louie-apple committed:
Public Release 2.2.15 

- Various ORKTextChoiceOther and ORKChoiceOtherViewCell improvements 
- Bumped version to 2.2.15
- Bumped Cocoapod version to 2.2.15

louie-apple committed:
Public Release 2.2.15

Various ORKTextChoiceOther and ORKChoiceOtherViewCell improvements
Bumped version to 2.2.15
Bumped Cocoapod version to 2.2.15

louie-apple committed:
Merge branch 'stable' into main

Pariecemckinney-apple committed:
Public Release 3.0

cbaker6 committed:
ci: Add code integration testing to repo (#1569)

Pariecemckinney-apple committed:
Public release 3.0.1

louie-apple committed:
cleaning up unused strings (#1578)

aplummer-apple committed:
Set “Enable Module Verifier” flag to “Yes” only in Release build configuration (#1579)

Pariecemckinney-apple committed:
Update README.md

Pariecemckinney-apple committed:
Public release 3.1.0

simon-apple committed:
Added issue templates. (#1586)

Pariecemckinney-apple committed:
Public release 3.1.1

lukaskollmer committed:
fix crash when answering form question (#1599)

* fix crash when answering last question in a form, but there's subsequent informational (non-question) sections

* Update ResearchKitUI/Common/Step/Form Step/ORKFormStepViewController.m

* incorporate PR feedback

* update docs

sofia-apple committed:
Public Release 3.2.0

adonawa committed:
feat: Use main bundle localized key/values first if available (#1602)

sofia-apple committed:
Update installation instructions in README.md (#1608)

Domik234 committed:
Fix incorrect parameter passed to activeStepDuration in tremorTestTaskWithIdentifier (#1616)

Pariecemckinney-apple committed:
Public Release 3.4.0
